site/topics/parameters/tuning.rst

```diff
@@ -7,6 +7,39 @@ Tuning Hyperparameters
 Symbolic vs. Numeric Hyperparameters
 ====================================
 
+* Symbolic hyperparameters are the modular portions of the algorithms
+
+    * Crossover operator
+    * Selection strategy
+
+
+* Numeric hyperparameters are the numerical values that can be tuned
+
+    * Crossover rate
+    * Tournament size
+
+
+* Regardless of the hyperparameter type, their values need to be set
+
+    * A crossover rate needs to be set
+    * A decision needs to be made on which selection to use
+
+        * Symbolic hyperparameters may introduce additional numeric hyperparameters
+        * For example, if tournament selection is selected, the tournament size needs to be specified
+
+
+* Typically, symbolic hyperparameters are more high-level
+
+    * They make up the evolutionary computation algorithm
+
+
+* The numerical hyperparameters are low-level
+
+    * Specifies the details of the algorithm's variant a
+
+
+* An *evolutionary computation algorithm instance* is one that has all the necessary hyperparameters set
```

Comment:
read: you need to apply all arguments to a function for it to fully be applied all of it's arguments